package TryCatch;

public class TryCatchFinally {
public static void main(String[] args) {
//	case1
	try {
		
	} catch (Exception e) {
		
	}
//	Case2
	try {
		
	} catch (ArithmeticException e) {
		// TODO: handle exception
	}
	catch (Exception e) {
		// TODO: handle exception
	}
//	Case3
	try {
		
	} catch (Exception e) {
		// TODO: handle exception
	}
	catch (Exception e) {
		// TODO: handle exception
	}
//	Case4
	try {
		
	} 
	catch (Exception e) {
		// TODO: handle exception
	}
	finally {
		// TODO: handle finally clause
	}
//	case5
	try {
		
	} catch (Exception e) {
		// TODO: handle exception
	}
	try {
		
	} catch (Exception e) {
		// TODO: handle exception
	}
//	case6
	try {
		
	} finally {
		// TODO: handle finally clause
	}
//	case7
	try {
		
	}
//	case8
	catch(Exception e){
		
	}
//	case9
	finally {
		
	}
//	case10
	try {
		
	} finally {
		// TODO: handle finally clause
	}
	catch (Exception e) {
		// TODO: handle exception
	}
//	case11
	try {
		
	}
	try {
		
	} catch (Exception e) {
		// TODO: handle exception
	}
	finally {
		
	}
//	case12
	try {
		
	} catch (ArithmeticException e) {
		// TODO: handle exception
	}
	catch (Exception e) {
		// TODO: handle exception
	}
	finally {
		
	}
//	case13
	try {
		
	} catch (Exception e) {
		// TODO: handle exception
	}
	finally {
		
	}
	finally {
		
	}
//	case14
	try {
		
	} 
	System.out.println("hii");
	catch (Exception e) {
		// TODO: handle exception
	}
//	case15
	try {
		
	} catch (Exception e) {
		// TODO: handle exception
	}
	System.out.println("Hii");
	catch (Exception e) {
		// TODO: handle exception
	}
//	case16
	try {
		
	} catch (Exception e) {
		// TODO: handle exception
	}
	System.out.println("Hii");
	finally {
		
	}
//	case17
	try {
		try {
			
		} catch (ArithmeticException e) {
			// TODO: handle exception
		}
	} catch (Exception e) {
		// TODO: handle exception
	}
//	case18
	try {
		try {
			
		} catch (Exception e) {
			// TODO: handle exception
		}
		
	} catch (Exception e) {
		// TODO: handle exception
	}
	//case19
	try {
		try {
			
		}
	} catch (Exception e) {
		// TODO: handle exception
	}
//	case20
	try {
		
	} catch (Exception e) {
		// TODO: handle exception
		try {
			
		} catch (Exception e2) {
			// TODO: handle exception
		}
	}
//	case21
	try {
		
	} catch (Exception e) {
		// TODO: handle exception
	}
	finally {
		try {
			
		} catch (Exception e) {
			// TODO: handle exception
		}
		finally {
			System.out.println("hii");
		}
	}
//	case22
	try 
	System.out.println("try");
	catch (Exception e)
	System.out.println("catch");
	finally 
	System.out.println("Finnaly");
			
}
}
